bundles/pom.xml

```diff
@@ -1,553 +1,1083 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Can you revert the formatting of this file?

Reply:
Will be available with next push. I copied the file from addon repository.